### PR TITLE
Create encodefor.json - RE: Issue 1072

### PR DESCRIPTION
**Pull request by shawnoden**

### CODE REVIEW

**Comment by pfreitag** [comment]
Thanks!

**Comment by shawnoden** [comment]
@pfreitag  What did the failure mean? Do I need to add the file somewhere else too?

**Comment by pfreitag** [comment]
Yeah that was just because it was not in the index... I will run utilities/indexer.cfm and it will take care of it.

File: data/en/encodefor.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+	"name":"encodeFor",
+	"type":"function",
+	"syntax":"encodeFor(type, value)",
+	"returns":"string",
+	"related":["encodeForHTML", "canonicalize"],
+	"description":"Encodes a given string for safe output in the specified context. The encoding in meant to mitigate Cross Site Scripting (XSS) attacks.",
```
Comment (KamasamaK) [inline data/en/encodefor.json]: I think this is supposed to say "encoding _is_ meant to"

Comment (shawnoden) [reply data/en/encodefor.json]: PR #1077 issued.

**Pull request merged**